API/AddUser.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+	$inData = getRequestInfo();
+
+	$servername = "localhost";
+	$username = "root";
+	$password = "";
+	$dbname = "myDB";
+
+	// Create connection
+	$conn = new mysqli($servername, $username, $password, $dbname); 	
+	if( $conn->connect_error )
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		if userExists($conn, $inData['userName'])
+		{
+			returnWithError("Username Already Taken")
+		}
+		else
+		{
+			$hashed_password = password_hash($inData["password"], PASSWORD_DEFAULT);
+			
+			$stmt = $conn->prepare("INSERT INTO Users (ID, firstName, lastName, userName, password, dateCreated) VALUES (NULL, ?, ?, ?, ?, NULL)");
+			$stmt->bind_param("ssss", $inData["firstName"], $inData["lastName"], $inData["userName"], $inData["password"]);
+			$stmt->execute();
+			$result = $stmt->get_result();
+		}
+
+		$stmt->close();
+		$conn->close();
+	}
+
+	function userExists($conn, $username) {
+		$stmt = $conn->prepare("SELECT * FROM Users WHERE userName = ?;");
```

Comment:
Don't really need ";" in statement
```suggestion
		$stmt = $conn->prepare("SELECT * FROM Users WHERE userName = ?");
```